### PR TITLE
Update LICENSE file

### PR DESCRIPTION
**Pull request by hector-cao**
Move to GPLv3 to be compliant with Canonical licensing policy

### CODE REVIEW

**Review by bktan8 — APPROVED** [review]
LGTM

File: attestation/check-production.sh [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# Copyright (C) 2024 Canonical Ltd.
+#
+# This file is part of tdx repo. See LICENSE file for license information.
```
Comment (frenchwr) [inline attestation/check-production.sh:3]: Should we write something more specific in the two sentences at the bottom? Or has this wording already been developed/approved by the powers at be? I'm thinking something like the following: "This file is part of Canonical's TDX GitHub repository (https://github.com/canonical/tdx). See the LICENSE file in the repository for license information."

**Review by chrisschnabel — CHANGES_REQUESTED** [review]
1. The copyright date should be the date the file was first released. I believe that this is 2023 in most cases?
2. I agree with @frenchwr regarding specifying Canonical TDX 
3. It should include the boilerplate about warranty, etc...

I think I've generally addressed with this text, provided all Copyright are 2023:

This file is part of Canonical's TDX repository, available at https://github.com/canonical/tdx, which includes tools to setup and configure a confidential computing environment based on Intel TDX technology. See the LICENSE file in the repository for license information.

Copyright 2023 Canonical Ltd.

This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License version 3, as published by the Free Software Foundation.

This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranties of MERCHANTABILITY, SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.

You should have received a copy of the GNU General Public License along with this program.  If not, see <http://www.gnu.org/licenses/>.

**Comment by hector-cao** [comment]
> This file is part of Canonical's TDX repository, available at https://github.com/canonical/tdx, which includes tools to setup and configure a confidential computing environment based on Intel TDX technology. See the LICENSE file in the repository for license information.
> 
> Copyright 2023 Canonical Ltd.
> 
> This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License version 3, as published by the Free Software Foundation.
> 
> This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranties of MERCHANTABILITY, SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
> 
> You should have received a copy of the GNU General Public License along with this program. If not, see http://www.gnu.org/licenses/.

Thanks @chrisschnabel 
1) For the year, i think that is fine to use 2024 since this is for 24.04 release
2) Ok for the boilerplate but i m wondering if putting the link to GH canonical is a good idea, that will create an unnecessary dependancy that will need to be maintained for the long run

What do you think ?

File: guest-tools/image/create-td-image.sh [hunk 2]
```diff
@@ -1,3 +1,10 @@
+# This source code is a modified copy of https://github.com/intel/tdx-tools.git
+# See LICENSE.apache file for license information.
```
Comment (zvr-intel) [inline guest-tools/image/create-td-image.sh:2]:
```suggestion
# See LICENSE.apache file for original license information.
```

**Comment by zvr-intel** [comment]
I would also suggest you add an SPDX license identifier line.

Taking all previous discussion into account, the header could look like:

```
# This file is part of Canonical's TDX repository which includes tools
# to setup and configure a confidential computing environment
# based on Intel TDX technology.
# See the LICENSE file in the repository for the license text.

# Copyright 2024 Canonical Ltd.
# SPDX-License-Identifier: GPL-3.0-only

# This program is free software: you can redistribute it and/or modify it
# under the terms of the GNU General Public License version 3,
# as published by the Free Software Foundation.
# This program is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranties
# of MERCHANTABILITY, SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR PURPOSE.
# See the GNU General Public License for more details.
```

**Comment by chrisschnabel** [comment]
@hector-cao the changes that @zvr-intel proposes are also good ones: i.e. add 'original' when pointing to the Apache license; and add SPDX license identifier line.

**Pull request merged**